met/src/basic/vx_cal/is_leap_year.cc

```diff
@@ -91,13 +93,15 @@ void increase_one_month(int &year, int &month) {
 unixtime add_to_unixtime(unixtime base_unixtime,
     int sec_per_unit, double time_value, bool no_leap) {
   unixtime ut;
-  bool debug = false;
+  const char *ptr = getenv("MET_DEBUG");
```

Comment:
```suggestion
  
```
If using mlog, then this "const char *ptr = getenv("MET_DEBUG") " is no longer needed and can be removed.